pkg/controller/cluster/resource/resource.go

```diff
@@ -350,10 +350,34 @@ func StatefulSetForRack(r scyllav1alpha1.RackSpec, c *scyllav1alpha1.Cluster, si
 		},
 	}
 
+	sysctlContainer := sysctlInitContainer(c.Spec.Sysctls)
+	if sysctlContainer != nil {
+		sts.Spec.Template.Spec.InitContainers = append(sts.Spec.Template.Spec.InitContainers, *sysctlContainer)
+	}
 	sts.Spec.Template.Spec.Containers = append(sts.Spec.Template.Spec.Containers, *agentContainer(c))
 	return sts
 }
 
+func sysctlInitContainer(sysctls []string) *corev1.Container {
+	if len(sysctls) == 0 {
+		return nil
+	}
+	opt := true
+	return &corev1.Container{
+		Name:            "sysctl-buddy",
```

Comment:
Is this some kind of naming scheme here?

Reply:
Hehe. Any suggestions?

Reply:
sysctl-init? just sysctl

Reply:
No name?